src/components/ActivitiesList/ActivityList.tsx

```diff
@@ -0,0 +1,24 @@
+import ListView from "../ItemList/ItemList";
+import { Activity } from "./Definition";
+
+interface ActivityListProps {
+  activities: Activity[];
+  className?: string;
+}
+
+export default function ActivityList(props: ActivityListProps) {
+  return (
+    <ListView<Activity>
+      items={props.activities}
+      className={props.className}
```

Comment:
so for this one we won't pass in a className from props, but we can pass `styles.whatever` from the consumer component's css module
```suggestion
      className={styles.foo}
```

Reply:
Fixed! [4cb382a
](https://github.com/Code-the-Dream-School/ee-prac-team2-front/pull/21/commits/4cb382a2718c3ee68ce03b1834ecd8f181c3b202)